### DISCUSSION TITLE
Raised Back of Hand Medium-Light Skin Tone file is incomplete

### DISCUSSION THREAD

**Discussion by kennyspade** [Tarikul-Islam-Anik/Animated-Fluent-Emojis#4] [General]
It looks like the [Raised Back of Hand Medium-Light Skin Tone.png](https://github.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/blob/master/Emojis/Hand%20gestures/Raised%20Back%20of%20Hand%20Medium-Light%20Skin%20Tone.png) file wasn't completely uploaded. The file on Emojipedia is correct, so it should be possible to download from [https://emojipedia.org/microsoft-teams/1.0/raised-back-of-hand-medium-light-skin-tone/](https://emojipedia.org/microsoft-teams/1.0/raised-back-of-hand-medium-light-skin-tone/) and upload the complete file.

**Comment by kennyspade** [comment]
Similar issues with [Old Man Light Skin Tone.png](https://github.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/blob/master/Emojis/People%20with%20professions/Old%20Man%20Light%20Skin%20Tone.png) - [(Emojipedia link)](https://emojipedia.org/microsoft-teams/1.0/old-man-light-skin-tone/) and [Superhero Medium-Dark Skin Tone.png](https://github.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/blob/master/Emojis/People%20with%20professions/Superhero%20Medium-Dark%20Skin%20Tone.png) - [(Emojipedia link)](https://emojipedia.org/microsoft-teams/1.0/superhero-medium-dark-skin-tone/)

**Comment by Tarikul-Islam-Anik** [comment]
Hello, @kennyspade! 
Thanks for letting me know. I've re-uploaded the files. Here is the [**commit**](https://github.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/commit/b68086198dadf14080e7fcac2ce143993b2e4db0).
Have a great day! <img src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Smilies/Green%20Heart.png" alt="Green Heart" width="25" height="25" />
